extensions/ql-vscode/src/model-editor/model-evaluator.ts

```diff
@@ -107,8 +107,27 @@ export class ModelEvaluator extends DisposableObject {
   }
 
   public async openModelAlertsView() {
-    const view = new ModelAlertsView(this.app);
-    await view.showView();
+    const evaluationRun = this.modelingStore.getModelEvaluationRun(this.dbItem);
+    if (evaluationRun === undefined) {
+      return;
+    } else if (this.modelingStore.isModelAlertsViewOpen(evaluationRun)) {
```

Comment:
Do we need to go to the store for this or can we just do

```suggestion
    } else if (evaluationRun.isModelAlertsViewOpen) { 
```